tests/sysauth/sssd.pm

```diff
@@ -17,19 +17,33 @@ use warnings;
 use testapi;
 use utils;
 use version;
-
+use version_utils qw(is_sle is_opensuse);
 
 sub run {
     # Assume consoletest_setup is completed
     select_console 'root-console';
+    if (is_sle) {
+        assert_script_run 'source /etc/os-release';
+        if (is_sle '>=15') {
+            if (script_run('SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300) != 0) {
+                record_soft_failure 'bsc#1124318 - Fail to get PackageHub Pool Metadata - running the command again as a workaround';
+                assert_script_run 'SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300;
+                assert_script_run('SUSEConnect -p sle-module-legacy/${VERSION_ID}/${CPU}', 300);
```

Comment:
Here you _can_ use add_suseconnect_product, also, this would only happen if PackageHub failed the first time... what if it's fixed?...

Reply:
As always we need to remove softfail when it got fixed of course. At moment it is happens each time.

Reply:
I'm taling about L31

Reply:
updated now with new verification test run, thanks!